model/src/main/proto/example.proto

```diff
@@ -9,6 +9,10 @@ message UserAppHistory {
   bool already_opened_app = 1;
 }
 
+message OnBoardingFlow {
```

Comment:
Is this an example proto? It seems like we want to surface to the UI, so it should go in a final location. Also, please add documentation for this proto & its field.

Reply:
onboarding proto is created.